seeding/run-seeding.js

```diff
@@ -0,0 +1,52 @@
+const { loader } = require("./utilities/loader");
+const {
+  checkIfSeedWasExecuted,
+  markSeedAsExecuted,
+} = require("./utilities/real-time-db");
+const { seeds } = require("./seeds/seeds");
+const { exit } = require("process");
+const { initializeApp } = require("./config");
+
+initializeApp();
+
+const checkAndExecuteSeed = async (seed) => {
+  loader.start(`Checking if ${seed.description} was executed...`);
+  const isExecuted = await checkIfSeedWasExecuted(seed.key);
+  loader.stop();
+  if (!isExecuted) {
+    await seed.execute();
+    loader.start("Updating seed status...");
+    await markSeedAsExecuted(seed.key);
+    loader.stop();
+    await checkAndExecuteSeed(seed);
```

Comment:
why do we need to call this? we're marking it as executed, so why do we have to check if we've done it?